try {
    console.log(a);
} catch(err) {
    console.log(err.toString());
    console.log(typeof err);
}
console.log('111');
